language/move-lang/src/parser/syntax.rs

```diff
@@ -257,65 +282,80 @@ fn parse_module_name(tokens: &mut Lexer) -> Result<ModuleName, Error> {
 //      ModuleIdent = <Address> "::" <ModuleName>
```

Comment:
It looks like this should be: `ModuleIdent = <LeadingNameAccess> "::" <ModuleName>`